apps/mi/mi-azure-functions/image-policy.yaml

```diff
@@ -14,22 +14,6 @@ spec:
 ---
 apiVersion: image.toolkit.fluxcd.io/v1beta2
 kind: ImagePolicy
-metadata:
-  name: mi-azure-functions-dev
```

Comment:
Not in use as each PR would overwrite the existing image on dev whenever a build occurs, breaking the environment for the current user.
Prefer to deploy custom instance of mi-azure-functions to test against in dev.